tests/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ file(GLOB files
         "strings2/*.cpp"
         "strings/*.cc")
 
-add_executable(Tests ${files})
+add_executable(Tests ${files} stl/equals_check.cpp)
```

Comment:
You don't need to add this file manually. just add "stl/*.cpp" above so that all the stl tests are included as well